### DISCUSSION TITLE
How to disable mouse-left-hold event to prevent changing the order of tabs by mouse?

### DISCUSSION THREAD

**Discussion by nyngwang** [nanozuki/tabby.nvim#113] [Q&A] [answered]
As title. I felt that annoying when I accidentally hold and drag a tab my mouse and my tabs went out of order.

**Comment by nanozuki** [comment, marked as answer]
When using neovim's built-in tag label, the single-click, and mouse-left-hold event handler will automatically add to that label. tabby.nvim cannot disable that now. I am considering adding a feature to define user customer event handler and support disabling that.

#115

**Reply by nyngwang** [reply]
Thanks for opening the issue.
